### ISSUE TITLE
example_commands.sh fails with core dump at heinz step

### ISSUE THREAD

**Issue by samleenz** [raphael-group/netmix#1]
hi, this software looks really interesting but i'm having trouble running the example script.
specifically, the heinz step will not run due to a core dump
i've logged an [issue](https://github.com/ls-cwi/heinz/issues/8#issuecomment-576527299) but was hoping you might have an idea as to what is causing the issue? 

I've used conda to build the environment for the dependencies (not sure if `{do}cplex` was necessary?) 

```
conda create -n netmix -c bioconda -c conda-forge -c IBMDecisionOptimization\
  heinz  \
  python=3.6 \
  numpy \
  scipy \
  h5py \
  htop \
  tmux \
  networkx \
  cplex \
  docplex
```

the error that is given is over in the issue on the heinz repo. 

running this on a centos 7 server

thanks for your help :-)

**Comment by matthewreyna** [comment]
Thanks for your question. Unfortunately, I'm not sure what's causing this issue, but I'd guess that there's an issue with the Bioconda recipe for CPLEX. We installed heinz from the heinz GitHub repository and installed the the Python dependencies using the Python virtual environment (`venv`), so we did not use conda.

Thanks for following up on the issue on the heinz GitHub repository! They know what they're doing, and more options for installing heinz will help everyone.
